getting-started/Getting-Started.md

```diff
@@ -0,0 +1,174 @@
+# Getting Started with OpenTelemetry JS
+This guide will walk you through the setup and configuration process for a tracing backend (in this case [Zipkin](https://zipkin.io), but [Jaeger](https://www.jaegertracing.io) would be simple to use as well), a metrics backend like [Prometheus](https://prometheus.io), and auto-instrumentation of NodeJS.
+
+
+1. [Tracing Your Application with OpenTelemetry](#tracing-your-application-with-opentelemetry)
+    1. [Setting up a Tracing Backend](#setting-up-a-tracing-backend)
+    2. [Trace Your NodeJS Application](#trace-your-nodejs-application)
+        1. [Install the required OpenTelemetry libraries](#install-the-required-opentelemetry-libraries)
+        2. [Initialize a global tracer](#initialize-a-global-tracer)
+        3. [Initialize and register a trace exporter](#initialize-and-register-a-trace-exporter)
```

Comment:
Maybe add sections for "**Create basic span**" and "**Basic configuration**".

1. [Install the required OpenTelemetry libraries](#install-the-required-opentelemetry-libraries)
2. [Initialize a global tracer](#initialize-a-global-tracer)
3. [Create basic span](#create-basic-span)
4. [Basic configuration](#basic-configuration)
    1. [Sampler](#sampler)
    2. [Scope Manager](#scope-manager)
    3. [Propagation etc](#propagation)
5. [Initialize and register a trace exporter](#initialize-and-register-a-trace-exporter)

WDYT?

Reply:
Are those in the scope of a getting started guide? I see that as a different advanced usage guide.

Reply:
An advanced usage guide would cover topics like:

1. Using other exporters
2. Generating a custom span
3. Getting trace ids for logging
4. Enabling and disabling plugins
5. Sampling
6. Scope managers
7. Propagation
9. Baggage